ext/cl_khr_command_buffer.asciidoc

```diff
@@ -378,6 +380,11 @@ cl_int clGetCommandBufferInfoKHR(
     size_t* param_value_size_ret);
 ----
 
+The following SVM entry points are included only from version 2.0, adding selected commands
+from Section 5.6:
+
+include::{generated}/api/protos/clCommandSVMMemcpyKHR.txt[]
+include::{generated}/api/protos/clCommandSVMMemFillKHR.txt[]
```

Comment:
I'm getting an asciidoctor warning building the specs that I think is caused by these two lines:

```
asciidoctor: WARNING: generated/api/protos/clCommandSVMMemcpyKHR.txt: line 4: id assigned to block already in use: clCommandSVMMemcpyKHR
asciidoctor: WARNING: generated/api/protos/clCommandSVMMemFillKHR.txt: line 4: id assigned to block already in use: clCommandSVMMemFillKHR
```

Basically, these generated snippets include a "label" for the API entry points to this means that they can only be included once.  If we want to support including them multiple times - here, for example, and in the description of each API - then we'll need to do something to the spec tooling to allow this.

For now I would recommend just describing each API "manually", as is done above.

Reply:
Fixed.